build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
     }
 
     dependencies {
-        classpath "com.android.tools.build:gradle:${ANDROID_GRADLE_VERSION}"
+        classpath 'com.android.tools.build:gradle:2.3.1'
```

Comment:
I would much rather use the gradle version defined in the `gradle.properties` file. Could we update the version there, instead?